src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## **v2.3.18** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.3.18) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.3.18) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.3.18) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.3.18) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/2.3.18)
```

Comment:
>**v2.3.18 [](start = 3, length = 9)

Let's restart versioning at 2.4.0 due to all the breaking changes coming..

Reply:
Done.

---
In reply to: [559733791](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559733791) [](ancestors = 559733791)